app/Models/User.php

```diff
@@ -869,25 +863,64 @@ public function sumOfActiveFreePages(): int
     /* Transaction related */
 
     /**
-     * Returns the payed kkt amount in the semester. 0 if has not payed kkt.
+     * Pay kkt and netreg.
+     * Also updates workshop balances
+     * and the internet access expiry date.
+     * Returns an array with the two transaction objects
+     * and the new expiry date.
+     */
+    public function payKKTNetreg(int $kkt_amount, int $netreg_amount)
+    {
+        // Creating transactions
+        $kkt = Transaction::create([
+            'checkout_id' => Checkout::studentsCouncil()->id,
+            'receiver_id' => Checkout::studentsCouncil()->handler_id,
```

Comment:
the receiver can be any collegist that the student council delegates to, not necessarily the handler. In both cases.

Reply:
Despite how they do this nowadays.

Reply:
I now added it as a parameter.